packages/data-service/src/data-service.ts

```diff
@@ -749,45 +838,39 @@ export class DataServiceImpl implements DataService {
     }
   }
 
+  @op(mongoLogId(1_001_000_178), ([dbName, collName], result) => {
+    return { ns: `${dbName}.${collName}`, ...(result && { result }) };
```

Comment:
I left all the additional metadata we log as-is, but tbh sometimes it can be quite big, like in this case for example, which would also include all index detals, that can be very verbose, not sure I want to take any action here, but thought it's worth highlighting